monai/bundle/workflows.py

```diff
@@ -194,11 +196,12 @@ class ConfigWorkflow(BundleWorkflow):
 
     """
 
+    @deprecated_arg_default("workflow", None, "train", since="1.3", replaced="1.4")
```

Comment:
I don't see you changed any thing for the arg "workflow".

Thanks.

Reply:
My intention was that "train" is more commonly used, so I changed the default values for workflow; I'm not sure if you have any other concerns here.

Reply:
In which line you changed it? I didn't any change in current code:
<img width="598" alt="image" src="https://github.com/Project-MONAI/MONAI/assets/27533405/96ba5d96-fd05-4e11-87bb-596f493a392f">